README.md

```diff
@@ -1151,13 +1151,14 @@ Bear in mind that with the introduction of the new V8 engine alongside the new E
 
 <br /><br /><br />
 
-## ![✔] 8.8. Use multistage builds
+## ![✔] 8.8. Use multi-stage builds
```

Comment:
I believe that every reader wishes to know WHY soon. Without this, how can I tell if this topic is of interest? For example, 'Use multi-stage builds to producer leaner and safer images'

cc @js-kyle